test/collection_test.js

```diff
@@ -764,7 +764,7 @@ describe("Collection", () => {
         {id: id_3, title: "art3"},   // to be created
         {id: id_4, deleted: true},   // to be deleted
         {id: id_6, deleted: true},   // remotely deleted & missing locally, skipped
-        {id: id_7, title: "art7-b"}, // remotely conflicting
+        {id: id_7, title: "art7-b"}, // remotely updated
```

Comment:
Did something actually change in this test or was this just an imprecise comment?